ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Clang_64bit/tasks/main.yml

```diff
@@ -3,25 +3,29 @@
 # LLVM/Clang 64bit #
 ####################
 # Clang 64bit is an OpenJ9 prerequisite.
+# The original source of Clang64 was https://releases.llvm.org/7.0.0/LLVM-7.0.0-win64.exe being executed with the /S parameter
 
 - name: Test if Clang 64bit is already installed (required by OpenJ9)
   win_stat:
-    path: 'C:\Program Files\LLVM\bin\clang.exe'
+    path: 'C:\Program Files\LLVM64\bin\clang.exe'
   register: clang_64bit_installed
   tags: clang_64bit
 
 - name: Download Clang 64bit
   win_get_url:
-    url: https://releases.llvm.org/7.0.0/LLVM-7.0.0-win64.exe
+    url: https://ci.adoptopenjdk.net/userContent/winansible/llvm-7.0.0-win64.zip
     dest: 'C:\temp\'
     force: no
     checksum: 74b197a3959b0408adf0824be01db8dddfa2f9a967f4085af3fad900ed5fdbf6
     checksum_algorithm: sha256
   when: not clang_64bit_installed.stat.exists
   tags: clang_64bit
 
-- name: Install Clang 64bit
-  win_command: 'c:\temp\LLVM-7.0.0-win64.exe /S'
+- name: Install (unzip) Clang 64bit
+  win_unzip:
```

Comment:
I would be concerned about using `win_unzip`. The module requires `PSCX` to be installed which can be done using the `win_psmodule`, however that requires further installation of Powershell modules to work, which I couldn't get working with our vagrantfiles. 
https://docs.ansible.com/ansible/latest/modules/win_unzip_module.html#requirements

Reply:
I used `win_unzip` since some of the other roles in the windows playbook used `win_unzip`, like the `freemarker` and `Java_install`. Are there other ways of installing the `PSCX` module without using `win_psmodule`?

Reply:
Ah, my mistake. It may've just been the situation I was using it, but I haven't seen any issues with either of those roles before regarding it, so go ahead :-)